content/articles/getting-started-with-css/index.md

```diff
@@ -0,0 +1,194 @@
+# GETTING STARTED WITH CSS
+
+CSS is an abbreviation for cascading style sheets. It is a  language used to style the display of a document  written in a markup language like HTML. Alongside HTML and JavaScript, CSS is a foundation technology of the worldwide web.
+## Functions of CSS
+CSS helps developers change the presentation of web pages and make them responsive to different types of gadgets. It also helps them apply elements like lists and headings consistently throughout the site pages.
+
+In this article I will explain how to select elements using different styles,doing various manipulations to the elements selected, changing the font and width of an element and lastly, using CSS to style HTML.
+## 1.Selecting Elements
+### Using the Elements' Name
+Developers can select the element uing the element's tag for example `p`,`h2`,`h1` etc as I have shown in the example below.
+```html
+<style>
+   p{}
+</style>
+```
+`p` is the element's name which stands for paragraph.
+
+### Using a Class Name
+A class declaration is done by putting a fullstop before the class name in a style block for example `.red-text`then followed by a declaration block`{}`. Using a class name to select an element will be as shown below.
+```html
+<style>
+   .red-text{}
+</style>
+```
+
+
+### Using an Id Selector
+An element can be selected with a specific Id by writing a `#` character followed by the element's Id. Id attributes are only applicable to one element. An Id attribute has a higher dominance than a class declaration, therefore, prioritized if they collide.An example of an Id selector is like`#Properties-of-CSS`. It can be set in HTML like this;
```

Comment:
```suggestion
To use an elements id as a selector you prepend it with the `#` character followed by the element's Id. Id attributes are only applicable to one element. An Id attribute is more specific when compared to a class declaration, therefore, prioritized if they collide. You can read [more about specificity on CSS Tricks](https://css-tricks.com/specifics-on-css-specificity/). An example of an Id selector is like `#Properties-of-CSS`. It can be set in HTML like this;
```